### DISCUSSION TITLE
[FTL] Could not create runner: listen ip4:tcp : socket: operation not permitted

### DISCUSSION THREAD

**Discussion by 0xPugal** [projectdiscovery/naabu#302] [General]
```
litt1eb0yy@cloud:~$_ naabu -list targets.txt

                  __
  ___  ___  ___ _/ /  __ __
 / _ \/ _ \/ _ \/ _ \/ // /
/_//_/\_,_/\_,_/_.__/\_,_/ v2.0.6

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[INF] Running SYN scan with CAP_NET_RAW privileges
[FTL] Could not create runner: listen ip4:tcp : socket: operation not permitted
```

**Comment by ehsandeep** [comment]
@litt1eb0yy to avoid this error, you need to run naabu with root privileges.
It would be helpful if you could provide the information listed in the default issue template:
- Running OS
- Capabilities/Permissions granted to the binary
- Does the error happens sporadically or systematically at each run?

**Comment by storenth** [comment]
@ehsandeep Faced with same issue (after adding `naabu` v2.1.6 to the pipeline), invoked by the root user under Gitlab runner on Ubuntu with `libpcap-dev` onboard
<img width="1018" alt="ftl-runner-issue" src="https://user-images.githubusercontent.com/18244131/236428520-72cb27ff-3c93-4c5e-801a-b0fe5e2d9b76.png">

<img width="980" alt="ftl-naabu-issue" src="https://user-images.githubusercontent.com/18244131/236432831-a4210873-1cc8-44e0-afbf-fd0e5f7e16bf.png">

<img width="970" alt="ftl-naabu-health-check" src="https://user-images.githubusercontent.com/18244131/236437546-6b8b2fe4-0691-42dc-ba7f-89cd454b00d9.png">

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
